yt_dlp/extractor/zee5.py

```diff
@@ -114,3 +114,45 @@ def _real_extract(self, url):
             'episode_number': int_or_none(try_get(json_data, lambda x: x['index'])),
             'tags': try_get(json_data, lambda x: x['tags'], list)
         }
+
+
+class Zee5SeriesIE(InfoExtractor):
+    _VALID_URL = r'(?:https?://)(?:www\.)?zee5\.com/(?:global/)?(?:[^#/?]+/)?(?:(?:tvshows|kids)/[^#/?]+/)(?P<display_id>[^#/?]+)/(?P<id>[^#/?]+)/?(?:$|[?#])'
+    _TEST = {
+        'url': 'https://www.zee5.com/kids/kids-shows/krishna-balram/0-6-1871',
+        'playlist_mincount': 43,
+        'info_dict': {
+            'id': '0-6-1871',
+        },
+    }
+
+    def _entries(self, show_id):
+        video_ids = []
+        access_token_request = self._download_json(
+            'https://useraction.zee5.com/token/platform_tokens.php?platform_name=web_app',
+            show_id, note="Downloading access token")
+        headers = {
+            'X-Access-Token': access_token_request['token'],
+            'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.72 Safari/537.36',
+            'Referer': 'https://www.zee5.com/',
+        }
+        showurl = 'https://gwapi.zee5.com/content/tvshow/{}?translation=en&country=IN'.format(show_id)
+
+        seasonid = try_get(self._download_json(showurl, video_id=show_id, headers=headers), lambda x: x['season']['id'], str)
+        nexturl = 'https://gwapi.zee5.com/content/tvshow/?season_id={}&type=episode&translation=en&country=IN&on_air=false&asset_subtype=tvshow&page=1&limit=10'.format(seasonid)
```

Comment:
`on_air=false` - what happens if it is airing?

Reply:
this will work regardless

if `on_air=true` then the api returns the episodes in descending order (latest first) even if the show is not airing.